mapillary_tools/commands/process.py

```diff
@@ -244,31 +263,61 @@ def add_basic_arguments(self, parser: argparse.ArgumentParser):
         )
 
     def run(self, vars_args: dict):
-        if (
-            "geotag_source" in vars_args
-            and vars_args["geotag_source"] == "blackvue_videos"
-            and (
-                "device_make" not in vars_args
-                or ("device_make" in vars_args and not vars_args["device_make"])
-            )
-        ):
-            vars_args["device_make"] = "Blackvue"
-        if (
-            "device_make" in vars_args
-            and vars_args["device_make"]
-            and vars_args["device_make"].lower() == "blackvue"
-        ):
-            vars_args["duplicate_angle"] = 360
+        if vars_args["geotag_sources"]:
+            # gpx:format=A,exif:format=B -> [{source: 'gpx', 'format': 'A'}, {'source': 'exif', format': 'B'}]
```

Comment:
Move all the parsing logic to `process_geotag_properties.py`?

Reply:
Yep, that's part of "Wire them better" in the TODO list :) but I first wanted to define the options format. We can maybe discuss it tomorrow.